website/docs/developer/research/research-s3-bucket-policies.md

```diff
@@ -0,0 +1,156 @@
+---
+id: research-s3-practices
+title: S3 Future Direction/Best Practices
+description: Research notes on S3 best practices and some directions to take development.
+---
+
+## Overview
+
+Presently, our S3 buckets are only defined through [documentation](../deployment-guide/creating-orca-glacier-bucket.md) and there are changes we wish to make to support future development.
+This document aims to document the desired final state of our S3 buckets.
+
+## Versioned Glacier Bucket
+
+Note that this work depends on completing (ORCA-351)[https://bugs.earthdata.nasa.gov/browse/ORCA-351]
+
+Suggested name: PREFIX-orca-glacier-archive-versioned
+
+Our [current Glacier bucket instructions](../deployment-guide/creating-orca-glacier-bucket.md) are well suited to storage.
+There are a few changes we should consider.
+Presently, ORCA does not handle versioned data, but it also does not preclude that capability in its buckets.
+Setting versioning on Glacier allows for finer-grained data backup as users could recover from a specific version of a file being overwritten.
+To enable development towards this goal, we should either replace the existing bucket with a versioned bucket and move existing data over, or instruct users on how to [enable versioning on their buckets](https://docs.aws.amazon.com/AmazonS3/latest/userguide/manage-versioning-examples.html).
+
+We should also remove the ACL capabilities due to AWS potential deprecation.
+This will be detailed in a [future section](#acl-rule-replacement).
+
+## WORM Glacier Bucket
+
+Suggested name: PREFIX-orca-glacier-archive-worm
+
+Some data will never have additional versions. This includes raw or near-raw satellite data. This data should be stored in a Read Once Write Many (WORM) model, and be protected from accidental deletion, as it is the foundation of all derived data.
+
+Permissions for this bucket will likely be identical to the [Versioned Glacier Bucket](#versioned-glacier-bucket)
+
+The naming dichotomy between this and the [Versioned Glacier Bucket](#versioned-glacier-bucket) is slightly deceptive, as this bucket will be versioned as well.
+However, this is only to enable [Object Lock](https://aws.amazon.com/blogs/storage/protecting-data-with-amazon-s3-object-lock/)
```

Comment:
is there any additional cost to use this feature?

Reply:
No. It only modifies the behavior for deletion requests.